docs/AzDO-Example.md

```diff
@@ -0,0 +1,145 @@
+# Continuos Integration and Continuos Deployment using Azure DevOps
+
+The following instructions demonstrate how to deploy the contents of this example repository using Azure DevOps Repos and Azure DevOps Pipelines.
+
+## Pre-reqs
+
+To execute this solution you will need:
+
+- Azure DevOps Account
+- An Azure DevOps Repo configured ([how to configure an Azure DevOps Repo](https://docs.microsoft.com/en-us/azure/devops/repos/get-started/sign-up-invite-teammates?view=azure-devops))
+- Three API Management deployed (DEV, QA and PROD) ([how to deploy a Create a new Azure API Management service instance](https://docs.microsoft.com/en-us/azure/api-management/get-started-create-service-instance))
+- Some API(s) on your API Management ([Add an API manually](https://docs.microsoft.com/en-us/azure/api-management/add-api-manually))
+
+## Architecture
+
+![](images/Architecture.png)
+
+This is one example of how to use Azure DevOps Repo and Azure DevOps Pipelines to enable a Continuous Integration and Continuous Deployment (CI/CD) for API\'s on an Azure API Management Service. There're other ways to do this, but this is a \"Keep it Simple\" to help you start.(Planning to have another version using Github and Github Actions soon).
+
+This is our Development API management. In general, developers will create their API\'s in one instance of APIM to test it.
+
+Create your API's on API Management Development environment. ([How to create API's on API Management](https://docs.microsoft.com/en-us/azure/api-management/import-and-publish))
+
+![](images/apim-dev.png)
+
+## APIM DevOps Toolkit
+
+**Running the Extractor**
```

Comment:
Would be great to mention that in this example we are using the Extractor to create the ARM templates. Customers can also use the Creator tool.